LiteFarmOrg/LiteFarm
# LF2230/allow completion of tasks for psuedo users

This PR adds functionality to complete tasks for pseudo users.

To test: 
1. Create a pseudo user on your farm by adding a user with no email
2. Assign a task to the pseudo user
3. Verify that for non admin accounts e.g. farm worker they do not have the option to complete the task for the pseudo user
4. Complete the task from an admin account (Farm Owner, Farm manager, extension officer)
5. Verify that the task is completed properly
6. Repeat this for both a harvest task and a non harvest task
